src/main/java/frc/robot/subsystems/elevator/ElevatorIO.java

```diff
@@ -0,0 +1,30 @@
+package frc.robot.subsystems.elevator;
+
+import edu.wpi.first.units.Distance;
+import edu.wpi.first.units.MutableMeasure;
+
+public interface ElevatorIO {
+
```

Comment:
Create the instance of the inputs here, then you can have static access from the `Elevator` class, and the IO classes will have it as a member. Auto logged of course. Look at the example subsystem for reference.

Reply:
58c674daaaa4325d1d3a9a8d3f47ff476fa925d2

Reply:
Should be auto logged.

Reply:
08e0a400536838771a47de5a740c3fb4d9ca16a7